pr_agent/algo/utils.py

```diff
@@ -375,7 +375,7 @@ def get_user_labels(current_labels: List[str] = None):
 
 def get_max_tokens(model):
     settings = get_settings()
-    max_tokens_model = MAX_TOKENS[model]
+    max_tokens_model = MAX_TOKENS.get(model, 4000)
```

Comment:
**Suggestion:** The `get_max_tokens` function could be improved by handling the case where the model is not found in the `MAX_TOKENS` dictionary more gracefully. Instead of defaulting to `4000`, consider raising an exception or logging a warning. This will make it easier to debug issues related to missing models. [enhancement]
```suggestion
    max_tokens_model = MAX_TOKENS.get(model)
    if max_tokens_model is None:
        get_logger().warning(f"Model {model} not found in MAX_TOKENS dictionary. Defaulting to 4000 tokens.")
        max_tokens_model = 4000
```